week1/homework/src/server.js

```diff
@@ -5,16 +5,40 @@ const http = require('http');
 /* `createServer` MUST return an instance of `http.Server` otherwise the tests
  * will fail.
  */
+
 function createServer(port) {
   let state = 10;
+  function showResponseToClient(response, state) {
+    response.setHeader('Content-Header', 'application/json');
```

Comment:
Great work! 🎊 Really awesome that you have separated this logic into a separate function, maybe you could take it one step further and move the function completely out side of the `create server ` function?  Im also missing the status code. 
And if  you want to take it even one step further, you can use ex6 syntax and use arrow functions, you dont have, just a suggestion :)

Reply:
Done =)